freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/flow/FreeIpaPostInstallService.java

```diff
@@ -1,5 +1,14 @@
 package com.sequenceiq.freeipa.service.freeipa.flow;
 
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
```

Comment:
unnecessary re-ordering of imports

Reply:
why un-necessary? And moreover its due to settings from CDP and CB. I cant do much there.